lambda_src/drivers/process_https.py

```diff
@@ -98,10 +98,23 @@ def process_row(
     req_method: str = method.upper()
 
     if json:
+        
         req_data: Optional[bytes] = (
             json if json.startswith('{') else dumps(parse_header_dict(json))
         ).encode()
         req_headers['Content-Type'] = 'application/json'
+        if json.startswith('arn'):
+            data=(decrypt_if_encrypted(json))
+            data=loads(data)
+            json_host=data['host']
+            if json_host and req_host and json_host != req_host:
+                raise ValueError(
+                "Requests can only be made to host provided in the auth header."
+                )
+            if('clientId' and 'secret' in data):
```

Comment:
these are details of a single EF, so they shouldn't go into a GEFF driver, since those should be Generic